src/components/pages/CreateAIPage.js

```diff
@@ -0,0 +1,152 @@
+import React, { useState } from "react";
+import Navbar from "../reusable/NavBar";
+import Sidebar from "../reusable/SideBar";
+import { useNavigate } from "react-router-dom";
+import doggy from "../../assets/images/doggy.jpg";
+
+const CreateAIPage = () => {
+  const navigate = useNavigate();
+  const [investorName, setInvestorName] = useState(null);
+  const [showError, setShowError] = useState(false);
+  const [profitStop, setProfitStop] = useState(50);
+  const [lossStop, setLossStop] = useState(50);
+
+  /*
+  Function called when the user attempts to save changes. Will check all the user values and 
+  attempt to update them.
+  */
+  const saveChanges = () => {
+    if (
+      investorName === null ||
```

Comment:
Will has a check for length > 2 . @WillOrtiz7  should Jason do the same ?

Reply:
I'll just add it